src/containers/my-cooperations/cooperation-notes/create-or-edit-note/CreateOrEditNote.tsx

```diff
@@ -76,16 +69,20 @@ const CreateOrEditNote = ({
       await onSubmit(data)
     }
   })
+  if (userIsLoading || !userResponse) {
+    return <Loader size={20} />
+  }
+  const firstName = userResponse?.firstName
+  const lastName = userResponse?.lastName
+  const photo = userResponse?.photo
 
   const userPhoto = photo
     ? new URL(photo, import.meta.env.VITE_APP_IMG_USER_URL).href
     : undefined
   const isNameValid = Boolean(firstName && lastName)
   const userName = isNameValid && `${firstName} ${lastName}`
```

Comment:
After adding the loader condition, there is no case when `userResponse` will be `falsy`
```suggestion
  const userName = `${firstName} ${lastName}`
```